srcpkgs/solvespace/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'solvespace'
+pkgname=solvespace
+version=2.3
+revision=1
+wrksrc=solvespace
+only_for_archs="i686 x86_64 x86_64-musl"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release"
+hostmakedepends="pkg-config"
+makedepends="glew-devel gtkmm2-devel json-c-devel"
+short_desc="Parametric 2d/3d CAD"
+maintainer="Robert Lowry <bobertlo@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://solvespace.com/"
```

Comment:
https://solvespace.com/index.pl gets me an SSL link... not sure why https://solvespace.com redirects to http but oh well.